storybook/storybook-react/src/shared/argTypes.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+export const marginArgType = {
+  control: {
+    type: 'inline-radio',
```

Comment:
These horizontal radio buttons are a bit fiddly imo, any reason to choose them over vertical radio's?

Reply:
I like that their compactness allows more props in view, and that props with the same options appear related directly.

As we current only use vertical radios everywhere I’ll change these to match. Let’s revisit later.